python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -302,7 +303,9 @@ class RandomForestClassifier(Base):
     def __getstate__(self):
         state = self.__dict__.copy()
         del state['handle']
-        self.model_pbuf_bytes = self._get_model_info()
+        if self.n_cols:
+            # only if model has been fit previously
+            self.model_pbuf_bytes = self._get_model_info()
```

Comment:
Not for this PR, but I remember you raised something about renaming this function? `_get_model_info()` seems like not the best name for this

Reply:
yup, should rename in future RF change.